src/core/api/preview.ts

```diff
@@ -0,0 +1,38 @@
+import fetch from 'cross-fetch';
```

Comment:
In Next.js, you don’t need to use cross-fetch. Just use [fetch](https://nextjs.org/docs/app/api-reference/functions/fetch) directly.